tests/testthat/_snaps/derive_vars_query.md

```diff
@@ -3,16 +3,16 @@
     Code
       derive_vars_query(mutate(my_ae, AELLTCD = as.logical(AELLTCD)), query)
     Condition
-      Error in `derive_vars_query()`:
+      Error in `get_vars_query()`:
       ! The source variables (values of `SRCVAR`) must be numeric or character.
       i AELLTCD is of type logical
 
-# derive_vars_query Test 6: Error is given when both TERMCHAR/TERMNUM are NA or empty
+# derive_vars_for_query Test 6: Error is given if both TERMCHAR/TERMNUM are NA/empty
 
     Code
       derive_vars_query(my_ae, query)
     Condition
-      Error in `derive_vars_query()`:
+      Error in `get_vars_query()`:
```

Comment:
umm... @bundfussr this is that issue again where a function is returned with an error that the user didn't directly call.  Do you think we can update this to what you did in that big PR with all the abort stuff?

Reply:
Hm, I am not sure if this can be solved easily. We could put the `get_vars_query()` call in `derive_vars_query()` into a `tryCatch()` and issue the error with an updated `call` argument.

Reply:
added a tryCatch per this that adds a sentence about how the error is being thrown because get is called in derive - lmk what you think